### PR TITLE
Add `dedi_sendPrintsToClient` to server convars

### LINKED ISSUE #150: Document `dedi_sendPrintsToClient`
https://github.com/R2Northstar/NorthstarLauncher/pull/418

### PR DESCRIPTION
**Pull request by itscynxx**
Ngl, kind of just guessing and writing based off of https://github.com/R2Northstar/NorthstarLauncher/pull/418 as I don't have a way to test this myself (no access to ports), probably needs to have changes suggested by someone because of this

Closes #150

### CODE REVIEW

**Review by ASpoonPlaysGames — CHANGES_REQUESTED** [review]
Description makes sense, but from looking at the code and doing my own testing there are some inaccuracies in the values:

- `-1` -> no prints are sent to clients
- `0` -> prints are sent to the first client only
- `1` -> prints are sent to all clients

default value is -1 (no prints)

**Comment by itscynxx** [comment]
threw something together quickly because ive got class here in a second

File: docs/hosting-a-server-with-northstar/dedicated-server/README.md [hunk 1]
```diff
@@ -140,6 +140,7 @@ They allow the server admin to set server's properties like the name, port, and
...
 | `sv_max_snapshots_multiplayer`                   | The number of snapshots stored locally for use in replays, this should be set to sv\_updaterate\_mp \* 15                                                                                   | `300`                          | `int`                                              |
 | `host_skip_client_dll_crc`                       | Whether the server should allow clients with modified client.dll files to connect, these are used for visor colour edit mods                                                                | `1`                            | `0-1`                                              |
+| `dedi_sendPrintsToClient`                        | Whether the dedicated server should forward all logs to connected clients, which is useful for testing/debugging on a dedicated server. If -1, send no prints to clients. If 0, send prints to the first client connected only. If 1, send prints to all clients                                                                   | `-1`                            | `-1-1`                                             |
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/dedicated-server/README.md]:
```suggestion
| `dedi_sendPrintsToClient`                        | Whether the dedicated server should forward all logs to connected clients, which is useful for testing/debugging on a dedicated server. If `-1`, send no prints to clients. If `0`, send prints to the first client connected only. If `1`, send prints to all clients                                                                   | `-1`                            | `-1-1`                                             |
```

File: docs/hosting-a-server-with-northstar/dedicated-server/README.md [hunk 2]
```diff
@@ -140,6 +140,7 @@ They allow the server admin to set server's properties like the name, port, and
...
 | `sv_max_snapshots_multiplayer`                   | The number of snapshots stored locally for use in replays, this should be set to sv\_updaterate\_mp \* 15                                                                                   | `300`                          | `int`                                              |
 | `host_skip_client_dll_crc`                       | Whether the server should allow clients with modified client.dll files to connect, these are used for visor colour edit mods                                                                | `1`                            | `0-1`                                              |
+| `dedi_sendPrintsToClient`                        | Whether the dedicated server should forward all logs to connected clients, which is useful for testing/debugging on a dedicated server. If `-1`, send no prints to clients. If `0`, send prints to the first client connected only. If `1`, send prints to all clients                                                                   | `-1`                            | `-1-1`                                             |
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/dedicated-server/README.md:143]: Note sure if I wanna remove the sentence

> `which is useful for testing/debugging on a dedicated server`

to maintain table size of just keep it at the cost of technically having to resize table in source...

Comment (itscynxx) [reply docs/hosting-a-server-with-northstar/dedicated-server/README.md:143]: Do we want to remove this line? Don't feel like much is lost if we do, most people wanting to use this probably know what to use it for. Not really sure what `having to resize table in source` means

Comment (GeckoEidechse) [reply docs/hosting-a-server-with-northstar/dedicated-server/README.md:143]: Welp kinda overlooked that one. Just gonna merge as is for now I guess

**Review by GeckoEidechse — APPROVED** [review]
The table is kinda not properly formatted in source now but should be fine.

**Pull request merged**